site/en/docs/extensions/mv3/intro/index.md

```diff
@@ -1,51 +1,60 @@
 ---
 layout: 'layouts/doc-post.njk'
-
-# The page title. This appears at the top of the doc and as the page name
-# in Google Search.
 title: Welcome to Manifest V3
-
-# This appears below the title and is an optional teaser
 subhead: 'A step in the direction of security, privacy, and performance.'
-
-# This appears in the ToC of the project landing page at
-# /docs/[project-name]/. It also appears in the <meta description> used in 
-# Google Search.
 description: 'An introduction to Manifest V3'
-
-# The publish date
 date: 2020-11-09
+updated: 2022-09-02
+---
 
-# An optional updated date
-# updated: 2020-10-16
+This site introduces Manifest V3, short for Manifest Version 3, which is the latest iteration of the Chrome extension platform. It shares the background and reasons for introducing Manifest V3 and the vision for the platform's future, along with resources on how to migrate.
 
-# A list of authors. These usernames correspond to the keys in the
-# _data/authorsData.json file.
+## What is a Manifest?
 
----
+An extension [Manifest][doc-manifest] gives the browser information about the extension, such as
+the most important files and the capabilities the extension might use. The extension platform features change when there's a new [Manifest version][manifest-version].
+
+## Introducing Manifest V3 
+
+Manifest V3 represents one of the most significant shifts in the extensions platform since it
+launched a decade ago. Manifest V3 extensions will enjoy enhancements in security, privacy, and
+performance; they can also use more contemporary Open Web technologies such as service workers and
+promises. 
+
+## Manifest V3 resources
+
+Manifest V3 is part of a shift in the philosophy behind how we approach end-user security and privacy. The following articles provide an overview of Manifest V3, the reasons behind it, and how to
```

Comment:
```suggestion
Manifest V3 is part of a shift in the philosophy behind user security and privacy. The following articles provide an overview of Manifest V3, the reasons behind it, and how to
```

Reply:
A writing best practice is to omit needless words. Needless words are any that can be removed without changing the meaning of the sentence.

Reply:
Totally agree!! 💯